medusa/providers/torrent/json/danishbits.py

```diff
@@ -40,6 +40,7 @@ def __init__(self):
 
         # Miscellaneous Options
         self.freeleech = True
+        self.headers.update({'User-Agent': USER_AGENT})
```

Comment:
I think this won't have any effect. As the self.headers is used to update self.session.headers, before you get to here.
So you should directly update self.session.headers in stead.